Default GitIgnore Override

romain325 opened the pull request:
Override default gitignore for a DotNetCore adapted version
Fix #112

The value for the gitignore doesn't perfectly fit
Need Review to make an efficient one

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/jhipster/jhipster-dotnetcore?pullRequest=260) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/jhipster/jhipster-dotnetcore?pullRequest=260) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/jhipster/jhipster-dotnetcore?pullRequest=260) it.</sub>

nicolas63 commented:
you can look this file : (https://raw.githubusercontent.com/jhipster/jhipster-net-sample-app-template/master/.gitignore) 
you need this part 
```
#Angular
**/wwwroot/
**/ClientApp/dist/
```

You can also delete the gitignore in https://github.com/jhipster/jhipster-dotnetcore/blob/master/generators/common/index.js

romain325 commented:
I added different adapted .gitignore and removed the gitignore default in the index.js, I also noticed that the README.md was also copied from default and then overwritted by the dotnet specific 
I removed it and tested it, nothing changed --> need an approval to be sure i haven't done anything wrong

nicolas63 commented on generators/common/templates/dotnetcore/gitignore.ejs:
```diff
@@ -0,0 +1,293 @@
+.vs
+.vscode
+
+#Angular
+**/wwwroot/
+**/ClientApp/dist/
```
duplicated lines

nicolas63 commented on generators/common/templates/dotnetcore/gitignore.ejs:
```diff
@@ -0,0 +1,293 @@
+.fake/
+
+#IntelliJ
+.idea/
```
duplicated line

nicolas63 commented on generators/common/templates/dotnetcore/gitignore.ejs:
```diff
@@ -0,0 +1,293 @@
+*.publishsettings
+orleans.codegen.cs
+
+node_modules/
```
duplicated line

nicolas63 commented:
Looks good to me
Congratulation for your first PR in this project :clap:
Can you remove draft mode ?

romain325 commented:
Draft mode disabled!

nicolas63 commented:
merged, thank you @romain325
